torch/utils/cpp_extension.py

```diff
@@ -24,6 +24,24 @@
 
 IS_WINDOWS = sys.platform == 'win32'
 
+# Taken directly from python stdlib < 3.9
+# See https://github.com/pytorch/pytorch/issues/48617
+def _nt_quote_args(args: List) -> List:
+    """Quote command-line arguments for DOS/Windows conventions.
+
+    Just wraps every argument which contains blanks in double quotes, and
+    returns a new argument list.
+    """
+    # XXX this doesn't seem very robust to me -- but if the Windows guys
+    # say it'll work, I guess I'll have to accept it.  (What if an arg
+    # contains quotes?  What other magic characters, other than spaces,
+    # have to be escaped?  Is there an escaping mechanism other than
+    # quoting?)
+    for i, arg in enumerate(args):
+        if ' ' in arg:
+            args[i] = '"%s"' % arg
+    return args
```

Comment:
Since it's our own function that is expected to work only in python-3.6.2 or newer, why not use list comprehensions and f-strings?
```suggestion
    return [f'"{arg}"' if ' ' in arg else arg for arg in args]
```

Reply:
This is a direct lift and shift of the original function, wanted to keep it as close to the original as possible to avoid any backwards compat issues